modules/build/src/main/scala/scala/build/bsp/BspServer.scala

```diff
@@ -254,6 +259,55 @@ class BspServer(
 
   override def onBuildExit(): Unit = ()
 
+  // TODO: reload dependencies
+  // TODO: reload scalac & javac options
+  // TODO: fix IntelliJ run tasks not picking up new sources
+  override def workspaceReload(): CompletableFuture[Object] =
+    super.workspaceReload().thenApply[Object] { res =>
+      val buildTargetsToRecompile: List[b.BuildTargetIdentifier] =
+        for { // this is just a workaround hack to emulate workspace reloading until Bloop itself supports it
+          case (targetUri, targetName) <- buildTargetNamesByUri.toList
```

Comment:
NIT no need of `case` here
```suggestion
          (targetUri, targetName) <- buildTargetNamesByUri.toList
```